cmd/utils.go

```diff
@@ -105,6 +105,10 @@ const (
 	httpsScheme = "https"
 )
 
+func nopCharsetConverter(label string, input io.Reader) (io.Reader, error) {
```

Comment:
Add a comment on why do we have this....

Reply:
> Add a comment on why do we have this....

Done